ci-operator/step-registry/stackrox/automation-flavors/ocp-4-crun-e2e/stackrox-automation-flavors-ocp-4-crun-e2e-workflow.yaml

```diff
@@ -0,0 +1,48 @@
+workflow:
+  as: stackrox-automation-flavors-ocp-4-crun-e2e
+  steps:
+    pre:
+    - ref: stackrox-stackrox-begin
+    - ref: openshift-manifests-crun
```

Comment:
This is a lot of duplication just to get this one step injected to `pre`.
/cc @davdhacs